app/views/layouts/_header.html.erb

```diff
@@ -132,7 +132,7 @@
                       <a class="dropdown-item" href="/useremail">Email search</a>
                       <a class="dropdown-item" href="/wiki/locked">Locked pages</a>
                       <a class="dropdown-item" href="/spam">Spam moderation</a>
-                      <a class="dropdown-item" href="/people">Contributors</a>
+                      <a class="dropdown-item" href="/features">Features</a>
```

Comment:
```suggestion
                      <a class="dropdown-item" href="/people">Contributors</a>
                      <a class="dropdown-item" href="/features">Features</a>
```

Thanks for opening this up @RATED-R-SUNDRAM 
Just a slight correction, the Contributors link should not be removed...the Features link should be added...sorry for any confusion caused by the issue...